screens/locationData.js

```diff
@@ -7,16 +7,18 @@ import Icon from '../components/Icon';
 import colours from '../styles/colours';
 import fonts from '../styles/fonts';
 import { pushLocalLocations } from '../actions/locations';
-import { requestPushPermission } from '../actions/permissions';
 import { showPushDialog } from '../actions/settings';
 import FailedListItem from '../components/FailedListItem';
 import { getStats, getFailedLocations } from '../selectors/stats';
 
 const styles = StyleSheet.create({
   container: {
+    borderTopWidth: 1,
```

Comment:
Not really a fan of these borders. They cut it up in weird ways.

Reply:
Haha yeah, same. Totally open to ideas. Also showing it to my girlfriend to get her opinion.